wagtail_transfer/static_src/components/PageChooserWidget/index.js

```diff
@@ -18,25 +18,27 @@ export default function PageChooserWidget({ apiBaseUrl, value, onChange, chosenT
     return (
       <div className={classNames.join(' ')}>
         <div className="chosen">
-          <span className="title">{value.title}</span>
-          {chosenText}
-          <ul className="actions">
+          <div className="transfer title-wrapper">
+            <h3 className="transfer title">{value.title}</h3>
+            <h6 className="transfer subtitle">{chosenText}</h6>
+          </div>
+          <ul className="transfer actions">
             <li>
               <button
                 type="button"
-                className="button action-choose button-small button-secondary"
+                className="action-choose link"
                 onClick={onClickChoose}
               >
-                Change
+                CHANGE
```

Comment:
We usually capitalise strings like this using CSS (using `text-transform`)

Reply:
Aha, great. Moved to that now.